lib/assertions/exception.test.js

```diff
@@ -119,6 +119,34 @@ testHelper.assertionTests("assert", "exception", function(pass, fail, msg) {
         "if not passed arguments",
         "[assert.exception] Expected to receive at least 1 argument(s)"
     );
+
+    pass(
+        "when matcher regexp matches",
+        function() {
+            throw new TypeError("Aright");
+        },
+        /right/
+    );
+
+    fail(
+        "when matcher regexp does not match",
+        function() {
+            throw new TypeError("Aright");
+        },
+        /nope/
+    );
+
+    function Matcher() {
+        this.name = "TypeError";
+    }
+    Matcher.prototype.message = "Nope";
+    pass(
+        "when non own property of object does not match",
```

Comment:
What does this mean? Could you elaborate on this test case? What's this thing with non own?

Reply:
This is the test case that I had to add for coverage.

> I also had to add another test to verify the else case of the hasOwnProperty check for coverage.

The code branch I'm testing is the `else` of this conditional:

https://github.com/sinonjs/referee/blob/6e33bb03dad427305c9f41e4dc7335913112f964/lib/assertions/exception.js#L51-L56

Reply:
@fatso83 Do you have a suggestion for the test name to make this more clear?

Reply:
Nah, it seems ok today :p